app/src/main/java/edu/uc/willi6jd/mycaloriecounter/ui/main/foodView/FoodActivity.kt

```diff
@@ -90,30 +87,30 @@ private class foodCustomerAdapter(context: Context): BaseAdapter(){
             val rowMain = layoutInflater.inflate(R.layout.food_row, viewGroup, false)
 
             val nameTextView = rowMain.findViewById<TextView>(R.id.listFoodName)
-            var currentItem = "Name: " + food.get(position)
+            var currentItem = "Name: " + food[position]
             nameTextView.text = currentItem
 
             val calTextView = rowMain.findViewById<TextView>(R.id.listFoodCal)
-            var currentCal = "Calories: " + cal.get(position)
+            var currentCal = "Calories: " + cal[position]
             calTextView.text = currentCal
 
             val fatTextView = rowMain.findViewById<TextView>(R.id.listFoodFat)
-            var currentFat = "Fat: " + fat.get(position)
+            var currentFat = "Fat: " + fat[position]
             fatTextView.text = currentFat
 
             val proTextView = rowMain.findViewById<TextView>(R.id.listFoodPro)
-            var currentPro = "Pro: " + pro.get(position)
+            var currentPro = "Pro: " + pro[position]
             proTextView.text = currentPro
 
             return rowMain
         }
 
     }
     object FoodItem {
-        var name = "";
```

Comment:
There is no need to use ";" in Kotlin and it does not fit in with the other lines in here so I deleted them.

Reply:
I agree.